README.md

```diff
@@ -82,6 +83,7 @@ variables in the desired endian format.
 [bincode]:https://crates.io/crates/bincode
 [byteorder]:https://crates.io/crates/byteorder
 [bytes]:https://crates.io/crates/bytes
+[packed_struct]:https://docs.rs/packed_struct/0.3.0/packed_struct/
```

Comment:
All links are to crates.io. Could you change it too?

Reply:
Sure!